docker-compose.yml

```diff
@@ -60,6 +60,8 @@ services:
       - APPLICATION_TENANTS_0_ISSUER=${APPLICATION_TENANTS_0_ISSUER}
       - APPLICATION_TENANTS_0_NAME=${APPLICATION_TENANTS_0_NAME}
       - APPLICATION_TENANTS_0_DBSCHEMA=${APPLICATION_TENANTS_0_DBSCHEMA}
+      - APPLICATION_PAGINATION_LIMITDEFAULT=${LIMIT_DEFAULT}
```

Comment:
I'm not sure this will work without the `-` between limit and default

Reply:
in the documentation it is said that `-` must be removed and replaced like that

Reply:
indeed, it will also work (not a "must be removed", look at other definition in the same docker compose file).

Reply:
in the docker compose file there are no other examples of properties with `-` to bind with. Should I add the` _ ` then ?

Reply:
`SUBSCRIPTION_ENTITY-SERVICE-URL` maybe ?